<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upload_page</class>
 <widget class="QMainWindow" name="upload_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/images/背景图片.png);
border-radius:30px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>30</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
border:none;}

QPushButton:pressed{
padding-top:5px;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="OCR_res.qrc">
      <normaloff>:/icons/icons/minus-circle-fill.png</normaloff>:/icons/icons/minus-circle-fill.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>30</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
border:none;}

QPushButton:pressed{

padding-top:5px}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="OCR_res.qrc">
      <normaloff>:/icons/icons/close-circle-fill.png</normaloff>:/icons/icons/close-circle-fill.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;

border-radius: 20px;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="cam_pic">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;华文琥珀&quot;;
color: rgb(54,179,152);
</string>
      </property>
      <property name="text">
       <string>                照片显示</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}</string>
      </property>
      <property name="text">
       <string>下一步</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shoot_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}</string>
      </property>
      <property name="text">
       <string>开始上传</string>
      </property>
     </widget>
     <zorder>pushButton_3</zorder>
     <zorder>shoot_2</zorder>
     <zorder>cam_pic</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="cam_pic_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>301</width>
        <height>281</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;华文琥珀&quot;;
color: rgb(54,179,152);
</string>
      </property>
      <property name="text">
       <string>                照片显示</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shoot">
      <property name="geometry">
       <rect>
        <x>134</x>
        <y>240</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}</string>
      </property>
      <property name="text">
       <string>拍照</string>
      </property>
      <property name="icon">
       <iconset resource="OCR_res.qrc">
        <normaloff>:/icons/icons/237拍照.png</normaloff>:/icons/icons/237拍照.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>34</width>
        <height>34</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>180</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}</string>
      </property>
      <property name="text">
       <string>进入下一步</string>
      </property>
      <property name="icon">
       <iconset resource="OCR_res.qrc">
        <normaloff>:/icons/icons/保存并下一步.png</normaloff>:/icons/icons/保存并下一步.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pre_shoot">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}</string>
      </property>
      <property name="text">
       <string>打开摄像头</string>
      </property>
      <property name="icon">
       <iconset resource="OCR_res.qrc">
        <normaloff>:/icons/icons/卡口视频摄像头,监控,桌面摄像头,网络摄像头.png</normaloff>:/icons/icons/卡口视频摄像头,监控,桌面摄像头,网络摄像头.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <zorder>pushButton_7</zorder>
     <zorder>pre_shoot</zorder>
     <zorder>cam_pic_1</zorder>
     <zorder>shoot</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(54, 179, 152);
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}
</string>
    </property>
    <property name="text">
     <string>本地照片识别</string>
    </property>
    <property name="icon">
     <iconset resource="OCR_res.qrc">
      <normaloff>:/icons/icons/文件夹.png</normaloff>:/icons/icons/文件夹.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	
	font: 18pt &quot;华文琥珀&quot;;
	color: rgb(255, 255, 255);
    border-radius:20px;
    background-color: rgb(54,179,152);}
QPushButton:hover {
    
	background-color:rgb(133, 218, 201) ;
}
QPushButton:pressed {
    
	background-color: rgb(54,179,152);
}
</string>
    </property>
    <property name="text">
     <string>视频拍照识别</string>
    </property>
    <property name="icon">
     <iconset resource="OCR_res.qrc">
      <normaloff>:/icons/icons/237拍照.png</normaloff>:/icons/icons/237拍照.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/images/切换.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/粮食监管.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/粮食监管.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/花草.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/花草.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;华文琥珀&quot;;
color: rgb(54,179,152);
</string>
    </property>
    <property name="text">
     <string>选择功能</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_2</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <resources>
  <include location="OCR_res.qrc"/>
 </resources>
 <connections/>
</ui>
